GUAC_FRIENDS.md

```diff
@@ -0,0 +1,22 @@
+# Friends of GUAC
+
+This file contains a list of projects related to GUAC.
+If you are a project that uses GUAC or provides an integration for/to GUAC, please open a PR and add yourself to the list!
+
+List of awesome GUAC projects!
+- [guac-rs](https://github.com/dejanb/guac-rs): This library provides toolkit for working with Guac from Rust.
+- [GUAC s3 Collector](https://github.com/trustification/trustification/tree/main/exporter): The exporter service gets notifications when SBOM or VEX documents get uploaded to S3 bucket and then ingest them into Guac.
+
+
+## GUAC Community & Supported projects
+
+If you would like to have your project be part of the [GUAC Github organization](https://github.com/guacsec/), we currently have categories of projects:
+- **Community:** A project that is maintained by the community. It should have relevance to the core use of GUAC.
+- **Supported:** A core supported project of GUAC that is maintained by the communtiy and also by the GUAC maintainers. These project should be of a certain maturity and adds to the core use of GUAC. 
+
+
+### Process
+
+To become a community project, a project must be on the "Friends of GUAC" list. Please open up a PR with the title: "[proposal/community-project] PROJECT NAME" with the link to the repository you'd like to donate as a community project. Maintainers will vote on a decision to include the project as a community project.
+
+To become a supported project, a project must be an existing community project. Please open up a PR with the title: "[proposal/supported-project] PROJECT NAME" with the link to the repository you'd like to donate as a community project.
```

Comment:
any existing issue template we can copy from for this? Like the OpenSSF one @mlieberman85?

Reply:
can iterate on this once we get more interest in this. Good for now.